redbot/cogs/audio/core/events/lavalink.py

```diff
@@ -287,7 +287,9 @@ async def lavalink_event_handler(
                         embed = discord.Embed(
                             title=_("Track Error"),
                             colour=await self.bot.get_embed_color(message_channel),
-                            description="{}\n{}".format(extra.replace("\n", ""), description),
+                            description="{}\n{}".format(
+                                extra.get("message").replace("\n", ""), description
```

Comment:
The key is guaranteed to exist for TRACK_EXCEPTION (`.get()` way was also wrong because if the key would ever not be there, there would be a TypeError due to it trying to call `None.replace()`):
```suggestion
                                extra["message"].replace("\n", ""), description
```